integration-tests/scaffolding/extrinsicHelpers.ts

```diff
@@ -141,8 +142,11 @@ export class Extrinsic<T extends ISubmittableResult = ISubmittableResult, C exte
     public async fundOperation(source?: KeyringPair, nonce?: number): Promise<void> {
```

Comment:
Updated this to only fund the operation when it didn't have enough money to do it already. This saved 50+ transactions when running against Frequency Rococo.

Reply:
This is fantastic!